client/src/pages/calendars/permissions.tsx

```diff
@@ -29,25 +30,26 @@ export function Permissions({
     });
   };
   return (
-    <details>
+    <details style={{ margin: '16px 0 20px' }}>
       <summary>Granted add permission</summary>
       {list.map((p, i) => (
-        <input
+        <Input
           key={i}
           type="text"
           value={p}
           onChange={onChange(i)}
           disabled={isFetching || save.isLoading}
-          style={{ display: "block" }}
+          style={{ display: "block", marginBottom: "12px" }}
```

Comment:
if you  need to override some base styles wrap the component with styled method:

```
const EmailInput = styled(Input)`
/* your styles here */
`
```